prisma/migrations/20230209210535_init/migration.sql

```diff
@@ -0,0 +1,49 @@
+-- CreateTable
+CREATE TABLE `code_reviews` (
+    `id` INTEGER NOT NULL AUTO_INCREMENT,
+    `created_at` DATETIME(3) NOT NULL DEFAULT CURRENT_TIMESTAMP(3),
+    `updated_at` DATETIME(3) NOT NULL,
+    `status` VARCHAR(10) NOT NULL,
+    `pull_request_link` VARCHAR(128) NOT NULL,
+    `slack_permalink` VARCHAR(128) NULL,
+    `slack_thread_ts` VARCHAR(30) NULL,
+    `slack_msg_id` VARCHAR(40) NULL,
+    `slack_channel_id` VARCHAR(20) NULL,
+    `jira_ticket` VARCHAR(128) NULL,
+    `note` TINYTEXT NULL,
+    `user_id` INTEGER NOT NULL,
+
+    PRIMARY KEY (`id`)
+) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+
+-- CreateTable
+CREATE TABLE `code_review_relations` (
+    `created_at` DATETIME(3) NOT NULL DEFAULT CURRENT_TIMESTAMP(3),
+    `updated_at` DATETIME(3) NOT NULL,
```

Comment:
This will give info about when someone first claim / react to the PR.   The updateAt field is use to determine the last update.  Should be useful info if we need to analyze an estimate on how long when the reviewer first claimed and then approve/give more feedback, etc... Just standard data field for any records.

Btw, these SQL files are auto generated by model schema declared in schema.prisma.

Similar to Laravel migration
```
npx prisma migrate dev
```